### PR TITLE
fix: roadmap area match styleguide updates

### PR DESCRIPTION
**Pull request by djmcquillan**
fixes: #138 

- updates to overall layout and color schemes of the roadmap container

DEMO:
https://starmaps-ezxooakzy-ipfs-ignite.vercel.app/roadmap/github.com/ipfs/go-libipfs/issues/54#detail

### CODE REVIEW

File: components/roadmap-grid/Roadmap.module.css [hunk 1]
```diff
@@ -165,8 +178,8 @@ a.milestoneTitle {
 }
 
 .timelineHeaderLineWrapper {
-  background: revert !important;
-  border-top: 1px solid;
+  background: #fafafa;
```
[the remark this answers is not in the document]
Comment (SgtPooki) [reply components/roadmap-grid/Roadmap.module.css:181]: Also, note that the background of the timelineHeaderLineWrapper is not filling the width of the first row of the timeline headers properly (see the far right). Background set to red (changed from `revert`) for ease of visibility:

### full view: 
image

### "zoomed in":

image

File: components/roadmap-grid/today-marker.module.css [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .todayMarkerWrapper {
   position: absolute;
-  top: 24px;
-  height: calc(100% - 24px);
+  top: 0px;
+  height: calc(100% + 20px);
```
Comment (SgtPooki) [inline components/roadmap-grid/today-marker.module.css:4]: What issue is this addressing?

File: components/roadmap-grid/Roadmap.module.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (djmcquillan) [reply components/roadmap-grid/Roadmap.module.css:181]: the naming here might be a bit confusing but the timelineHeaderLineWrapper is actually targeting the tickmark container
so the actual date header is the correct #F9FCFF, but I can remove the reverts and clean up so it is clearer
Screen Shot 2023-01-30 at 1 25 35 PM

File: components/roadmap-grid/today-marker.module.css (hunk 2 above)
Comment (djmcquillan) [reply components/roadmap-grid/today-marker.module.css:4]: some of the updates caused today marker to be displayed incorrectly vertically, addressed in this update as it deals with overall styling but can create new issue to be tracked if needed

File: components/roadmap-grid/Roadmap.module.css [hunk 3]
```diff
@@ -64,11 +66,12 @@
 .item.group {
   background: revert;
   grid-row: 1 / -1;
-  font-weight: 600;
+  font-weight: 500;
   grid-column: 1 / span 2;
   grid-column-end: none;
-  background: white;
+  background: #ffffff;
```
[the remark this answers is not in the document]
Comment (djmcquillan) [reply components/roadmap-grid/Roadmap.module.css:72]: this is for the root issue separation [commented here](https://github.com/pln-planning-tools/Starmap/issues/138#issuecomment-1407174866) I went with white background for now but may need further discussion

**Review by SgtPooki — APPROVED** [review]

**Pull request merged**